### ISSUE TITLE
Avatar: example naming, badge, and active ring

### ISSUE THREAD

**Issue by smhigley** [microsoft/fluentui#23128]
Avatar has a few accessibility bugs showing up in storybook:

1. The first avatar in the default story needs to have an accName exposed
2. The avatars in the "Active" story should be given realistic names so that the active/inactive state is exposed separately from the name
3. The avatars in the "Active Appearance" story need to have their active state exposed (this would be a component change rather than a story change)
4. The badges in the badge story have an `aria-label` applied to a generic `<span>` element -- that prop should be removed from the story code
5. The badges need a role of `img` and to not be hidden so the accName is correctly calculated and not relying on browser author correction mechanisms.
